src/components/Pages/Admin/ToolbarEditor.vue

```diff
@@ -0,0 +1,461 @@
+                { name: 'Record_To_Spreadsheet', mode: 59, src: '/vendor/img/tool_icons/Mode_recordtospreadsheet.svg' },
+                { name: 'Attach_Detach_Point', mode: 67, src: '/vendor/img/tool_icons/Mode_attachdetachpoint.svg' },
+            ],
+        };
+    },
+
+    computed: {
+        ...mapGetters('users', {
+            teacher: 'current',
+        }),
+
+        toolbars() {
+            return this.teacher.toolbars;
+        },
+    },
+
+    methods: {
+        ...mapActions('users', {
+            patch: 'patch',
+        }),
+
+        handleChange() {
+            this.content = this.transformArrayToolbarToString();
+            this.$emit('input', this.content);
+
+            this.$log.debug('this.content', this.content);
+        },
+
+        deleteTool(x, y) {
+            this.lists[x].splice(y, 1);
+            this.handleChange();
+        },
+
+        transformArrayToolbarToString() {
+            let string = '';
+
+            for (let i = 0; i < this.lists.length; i += 1) {
+                if (this.lists[i].length === 0 && (i !== (this.lists.length - 1))) string += '|';
+                for (let j = 0; j < this.lists[i].length; j += 1) {
+                    if (this.lists[i][j].mode !== undefined) string += `${this.lists[i][j].mode} `;
+
+                    if (j === (this.lists[i].length - 1)) {
+                        if (i !== (this.lists.length - 1)) string += '|';
+                    }
+                }
+            }
+
+            this.$log.debug('Toolbar string is', string);
+            return string;
+        },
+
+        transformStringToArrayToolbar(string) {
+            const array = string.split('|');
+
+            for (let i = 0; i < array.length; i += 1) {
+                array[i] = Array.from(array[i].split(' '));
+                if (array[i].length > 0) {
+                    for (let j = array[i].length - 1; j >= 0; j -= 1) {
+                        const a = array[i][j];
+                        this.$log.debug(`Before a(${i},${j}) = ${a}`);
+                        if (a === '') array[i].splice(j, 1);
+                        else {
+                            array[i][j] = parseInt(a, 10);
+                            array[i][j] = this.icons.find(tool => tool.mode === array[i][j]);
+                        }
+                        this.$log.debug(`After a(${i},${j}) = ${array[i][j]}`);
+                    }
+                }
+            }
+
+            this.$log.debug('Array is', array);
+
+            return array;
+        },
+
+        saveToolbar(selectedToolbar) {
+            this.handleChange();
+
+            this.$log.debug(this.teacher);
+
+            const toolbars = this.teacher.toolbars;
+            let name;
+            let overwrite = false;
+
+            if (selectedToolbar.length > 0) {
+                // eslint-disable-next-line no-restricted-globals
+                if (confirm(`Overwrite selected toolbar? (${selectedToolbar[0]})`)) {
```

Comment:
Linter emits warnings on this line and several next:
```
frontend_1  | /app/src/components/Pages/Admin/ToolbarEditor.vue
frontend_1  |   354:21  warning  Unexpected confirm  no-alert
frontend_1  |   359:27  warning  Unexpected prompt   no-alert
frontend_1  |   392:26  warning  Unexpected prompt   no-alert
frontend_1  |   397:21  warning  Unexpected alert    no-alert
```
Also, prompts and confirmations should be done with custom made modals (like these provided with `sweetalert` which is already embedded in the project) instead of [built-in browser alerts](https://eslint.org/docs/rules/no-alert).